Model/Http/BoldClient.php

```diff
@@ -152,17 +144,32 @@ private function getHeaders(int $websiteId): array
 
     /**
      * Get request url.
-     *
-     * @param int $websiteId
-     * @param string $url
-     * @return string
      */
     private function getUrl(int $websiteId, string $url): string
     {
         $apiUrl = $this->config->getApiUrl($websiteId);
+
+        if (str_contains($apiUrl, 'bold.ninja')) {
+            $parseApiUrl = parse_url($apiUrl);
+            $scheme = $parseApiUrl['scheme'];
+            $host = $parseApiUrl['host'];
+            $path = $parseApiUrl['path'];
+            $tunnelDomain = ltrim($path, '/');
+            $baseApiUrl = $scheme.'://'.$host;
+
+            if ($url === 'shops/v1/info') {
+                $apiUrl = $baseApiUrl;
+            }
+
+            if (str_contains($url, 'checkout_sidekick')) {
+                $apiUrl = $baseApiUrl.'/sidekick-'.$tunnelDomain;
```

Comment:
This would only work for specific tunnel names which are structured a certain way. I'm not against it for a local solution, but we should add that to the readme most likely.

Reply:
Yes, I was thinking on that exactly when doing this. 
Since you are here, could you let me know if we have a readme for the sidekick that we instruct on the tunnel creation?

Reply:
We do have a readme right in the project root for Sidekick, so any specific instructions about how set things up could go in there. There's also a `how_to_install.md` file which includes instructions for required setup steps so if you feel it fits better there that works too.

Reply:
@BoldBrandonM created a documentation PR for sidekick: https://github.com/bold-commerce/checkout-sidekick/pull/76